tests/test_hex.py

```diff
@@ -0,0 +1,40 @@
+import sys
+from unittest.mock import Mock
+
+sys.modules["mujoco_py"] = Mock()  # mock mujoco
```

Comment:
This is going to remain unsorted. Mujoco cant be imported anywhere on in the github workflow because otherwise, it will complain because there is no mujoco key. Not sure if there is a better workaround.

Reply:
that makes sense!

I added an explicit sort ignore line for this
https://github.com/brentyi/opthex/pull/3/commits/9673ebdfb0dd5a6fb9fbb5c63564d67aee1c2e56